### PR TITLE
Fraction of friends for CPass1 now depends on coll system.

### PR DESCRIPTION
**Pull request by chiarazampolli**
- HI (PbPB/XeXE) --> 0.2
- pp --> 0.5
- pA/Ap --> 1.0

### CODE REVIEW

**Comment by chiarazampolli** [comment]
Pinging also @shahor02 , I could not add him as reviewr (I guess because he has no admin rights...)

**Comment by shahor02** [comment]
Looks ok.

File: DataProc/CPass1/main_recCPass1.C [hunk 1]
```diff
@@ -105,7 +105,11 @@ void main_recCPass1(const char *filename="raw.root",Int_t nevents=-1, const char
   }
 
   // All friends
-  rec.SetFractionFriends(.2);
+  TString collSystem(strcmp(gSystem->Getenv("ALIEN_JDL_LPMINTERACTIONTYPE")));
```
Comment (catalinristea) [inline DataProc/CPass1/main_recCPass1.C:108]: Ciao Chiara,

Looks good, except for the call for strcmp... 
Could you please remove it?
Thanks!

Yours,
Catalin.

Comment (chiarazampolli) [reply DataProc/CPass1/main_recCPass1.C:108]: You are right!!! Sorry :-)

Comment (catalinristea) [reply DataProc/CPass1/main_recCPass1.C]: Ciao Chiara,

Now it's perfect, thanks a lot!

Yours,
Catalin.

**Pull request merged**